docs/Basics/Working with ROIs.md

```diff
@@ -28,27 +28,45 @@ In the options parameter,`getRois()` has a `kind` option which tells what kind o
 
 :::
 
-<!-- Add  a comment about what the image is -->
-
+ImageJS not only works with simple processing but it also specializes in advanced analysis of ROIs in scientific images.  
 Here is an example of how to extract ROIs from a real image.
 
-<!-- Here goes a your source code along with annotations about the generated intermediate images (don't put the code that creates the painted images) -->
+```ts
+import { Image, fromMask } from 'image-js';
+
+//image gets grayscaled and blurred for mask extraction(image 1)
+const image = sourceImage.grey().blur({ width: 5, height: 5 });
+//mask gets extracted(image 2)
+const mask = image.threshold();
+//receives all the regions of interest(colored on image 3)
+const roiMap = fromMask(mask);
+const rois = roiMap.getRois({ kind: 'black' });
+
+for (const roi of rois) {
+  //gets Feret diameters for each ROI(colored on image 4)
+  const feret = roi.feret;
+}
+```
 
-<!-- Include here an image 4 sub-images with captions: The original image, the mask, the original image with the ROIs painted in blue, the original image with the feret diameters painted over it. -->
```

Comment:
What I meant is to photoshop the 4 images into 1 with some captions overlaying the images.